Assets/HoloToolkit/Input/Scripts/BasicCursor.cs

```diff
@@ -51,21 +51,42 @@ void Awake()
             cursorDefaultRotation = this.gameObject.transform.rotation;
         }
 
-        void LateUpdate()
+        protected virtual RaycastResult CalculateRayIntersect()
         {
-            if (GazeManager.Instance == null || meshRenderer == null)
+            RaycastResult result = new RaycastResult();
+            if (GazeManager.Instance == null)
+            {
+                if (!hasLoggedGazeManagerError)
+                {
+                    Debug.LogError("Must have a GazeManager somewhere in the scene.");
+                    hasLoggedGazeManagerError = true;
+                }
+                return result;
+            }
+            result.Hit = GazeManager.Instance.Hit;
+            result.Position = GazeManager.Instance.Position;
+            result.Normal = GazeManager.Instance.Normal;
+            return result;
+        }
+
+        protected virtual void LateUpdate()
+        {
+            if (meshRenderer == null)
             {
                 return;
             }
 
+            // Calculate the raycast result
+            RaycastResult rayResult = CalculateRayIntersect();
+
             // Show or hide the Cursor based on if the user's gaze hit a hologram.
-            meshRenderer.enabled = GazeManager.Instance.Hit;
+            meshRenderer.enabled = rayResult.Hit;
```

Comment:
CalculateRayIntersect() will still return the rayResult (it's a struct) with .Hit=false. CalculateRayIntersect will log an error if gazemanager is null (like before).